### PR TITLE
added container

### PR DESCRIPTION
**Pull request by stevenjon**
I have finished please review my code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,3 @@
 <h1>Hello world</h1>
+
+<h1> Welcome to club and making team work</h1>
```
Comment (Yusufdev1998) [inline index.html]: Please make team work text red

Comment (Yusufdev1998) [reply index.html]: Ok done

Comment (stevenjon) [reply index.html]: Ok i ill do

**Review by Yusufdev1998 — CHANGES_REQUESTED** [review]
please change

**Review by Yusufdev1998 — APPROVED** [review]

**Pull request merged**